### PR TITLE
add forbidden path

### PR DESCRIPTION
**Pull request by farukaydin**
Fixes https://github.com/spree/spree/issues/6386

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://farukaydin-redirect-unauthorized-access.spree-spree-9ed724b77adec04c49c2.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://farukaydin-redirect-unauthorized-access.spree-spree-9ed724b77adec04c49c2.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: core/lib/spree/core/controller_helpers/auth.rb [hunk 1]
```diff
@@ -66,7 +66,7 @@ def try_spree_current_user
         def redirect_unauthorized_access
           if try_spree_current_user
             flash[:error] = Spree.t(:authorization_failure)
-            redirect_to '/unauthorized'
+            redirect_to '/forbidden'
```
Comment (JDutil) [inline core/lib/spree/core/controller_helpers/auth.rb]: This should be spree.forbidden_path

Comment (alepore) [inline core/lib/spree/core/controller_helpers/auth.rb]: :+1: hardcoded paths are always bad

**Comment by JDutil** [comment]
Not sure why specs are failing on undefined method forbidden_path looks like that should work fine.

**Comment by damianlegawiec** [comment]
@priyank-gupta @Mafi88 @jitendra I think this would be a nice addition - maybe we can fix those specs?

**Comment by priyank-gupta** [comment]
@damianlegawiec :+1:

**Comment by Mafi88** [comment]
@damianlegawiec @priyank-gupta @jitendra :+1: 

Getting on it

**Comment by Mafi88** [comment]
Continued in #6991

**Pull request closed without merging**